gossip/minimal/goshawk.go

```diff
@@ -89,6 +89,13 @@ func NewGoshawk(ctx context.Context, cfg *configpb.GoshawkConfig, hc *http.Clien
 	if cfg.SourceLog == nil || len(cfg.SourceLog) == 0 {
 		return nil, errors.New("no source log config found")
 	}
+	bufSize := cfg.BufferSize
+	if bufSize <= 0 {
+		// Allow at least one buffered STH in the channel so context cancellation
+		// works (everything shuts down at the same time, so the fetcher might otherwise
```

Comment:
s/so/as/ ?

Reply:
I think it's correct as-is -- fetcher getting blocked is a consequence not a cause of everything getting shut down at the same time.